windows/sound.cpp

```diff
@@ -560,13 +560,10 @@ CSound::CSound ( void           (*fpNewCallback) ( CVector<int16_t>& psData, voi
     // in case we do not have a driver available, throw error
     if ( lNumDevs == 0 )
     {
-        throw CGenErr ( "<b>" + tr ( "No ASIO audio device (driver) found." ) + "</b><br><br>" +
-            tr ( "The " ) + APP_NAME + tr ( " software requires the low latency audio "
-            "interface ASIO to work properly. This is not a standard "
-            "Windows audio interface and therefore a special audio driver is "
-            "required. Either your sound card has a native ASIO driver (which "
-            "is recommended) or you might want to use alternative drivers like "
-            "the ASIO4All driver." ) );
+        throw CGenErr ( "<b>" + tr ( "No audio device with ASIO driver detected" ) + "</b><br><br>" +
+            tr ( "To use " ) + APP_NAME + tr ( ", you need to install an ASIO driver first. "

```

Comment:
I think the following is more 
```
tr ( "You need to install an ASIO driver before running" ) + " " APP_NAME + ". "
```

Reply:
Instead of 
``` You need to install an ASIO driver .... ```

Perhaps we can use
```Please install an ASIO driver ... ```


_You need to_, and moreso, _require_ can be read as demanding to the user.

Reply:
True.  "Thou shalt!" might have been going too far, so I held back from that... :)

Reply:
Updated